### DISCUSSION TITLE
What is the reason for this_ Resource file missing in _nuxt dir?  请问什么原因会导致发布后_nuxt下资源文件丢失？

### DISCUSSION THREAD

**Discussion by AbyssWang** [nuxt/nuxt#9488] [Questions (Nuxt 2)]
It's like this:

At present, I have two servers, one can directly connect to the Internet, the other intranet server can not directly connect to the Internet, so I need to forward through the Internet server for Internet connection access.

My project is deployed on the Internet server and can be accessed normally by using the domain name. When my project is deployed on the intranet server, the access is abnormal. I use the chrome developer panel to view the sources of the two and find that the_ All the resource folders in the “_nuxt" directory are lost, without fonts and img folder

---------------------------------------------------------------------------------------------------------------

情况是这样子的：
目前我有两台服务器，一台可以直接连接互联网，另一台内网服务器无法直接连接互联网，需要通过互联网服务器转发来进行互联网连接访问。
我的项目部署在互联网服务器上，使用域名可以正常访问，而当我的项目部署在内网服务器上的时候，访问异常，我使用Chrome开发者面板查看二者的Sources，发现内网服务器的"_nuxt"目录下的资源文件夹全部丢失，没有fonts，没有img，请问这个可能是什么原因造成的？

**Comment by danielroe** [comment]
It's difficult to know what the cause of this is without more info, but it sounds like a server misconfiguration - particularly given the extra information given [here](https://github.com/nuxt/nuxt.js/discussions/9477).

**Comment by azataiot** [comment]
这应该是服务器的配置错误造成的。 你是否在使用nginx？ 或者是在用内网穿透呢？因为你给出的场景不够具体，也就没有办法给你具体的解决方案， 但是我觉得无非就是配置问题吧。
